qcelemental/models/align.py

```diff
@@ -69,10 +70,10 @@ def align_coordinates(self, geom, *, reverse=False) -> Array:
             algeom = algeom.dot(self.rotation)
             algeom = algeom + self.shift
             if self.mirror:
-                algeom[:, 1] *= -1.
+                algeom[:, 1] *= -1.0
         else:
             if self.mirror:
-                algeom[:, 1] *= -1.
+                algeom[:, 1] *= -1.0
```

Comment:
that's officious

Reply:
Oh, yea. I need to turn off `-S` it will homogenize `'` vs `"` on all strings.